tutorials/graphrag/GraphRAGDemo.ipynb

```diff
@@ -200,11 +200,15 @@
     "## Comparing Document Search Methods\n",
     "\n",
     "TigerGraph GraphRAG provides multiple methods to search documents in the graph. The methods are:\n",
-    "- **Hybrid Search**: This method uses a combination of vector search and graph traversal to find the most relevant information to the query. It uses the selected algorithm to search the embeddings of documents, document chunks, entities, and relationships. These results serve as the starting point for the graph traversal. The graph traversal is used to find the most relevant information to the query.\n",
+    "- **Similarity Search**: This is the baseline, traditional vector search approach. It compares the user query against the embeddings of document chunks, and returns the closest top-k matches. While efficient, this method is limited to semantic similarity and does not capture additional context or structural connections.\n",
```

Comment:
"It compares the user query against the embeddings of document chunks" -> "It compares the query embedding against the document chunk embeddings"